WalletWasabi.Gui/ViewModels/ViewModelBase.cs

```diff
@@ -1,50 +1,53 @@
-using Newtonsoft.Json;
 using ReactiveUI;
 using System;
 using System.Collections;
-using System.Collections.Generic;
 using System.ComponentModel;
-using System.Linq;
-using System.Reflection;
-using WalletWasabi.Gui.ViewModels.Validation;
-using WalletWasabi.Models;
+using WalletWasabi.Gui.Validation;
 
 namespace WalletWasabi.Gui.ViewModels
 {
-	public class ViewModelBase : ReactiveObject, INotifyDataErrorInfo
+	public class ViewModelBase : ReactiveObject, INotifyDataErrorInfo, IRegisterValidationMethod
 	{
+		private Validations _validations;
+
 		public ViewModelBase()
 		{
-			var vmc = Validator.PropertiesWithValidation(this).ToList();
-			if (vmc.Count == 0)
-			{
-				return;
-			}
-
-			ValidationMethodCache = vmc;
+			_validations = new Validations();
+			_validations.ErrorsChanged += OnValidations_ErrorsChanged;
```

Comment:
Unsubscribe?

Reply:
not required, we own the object and its lifetime matches ours. GC will still be able to collect.